ironfish/src/webApi.ts

```diff
@@ -210,7 +210,8 @@ export class WebApi {
     return response.data
   }
 
-  async submitTelemetry(payload: { points: Metric[]; graffiti: string }): Promise<void> {
+  async submitTelemetry(points: Metric[], graffiti?: string): Promise<void> {
```

Comment:
Does it work if we leave the function signature mostly the same, but just make the field optional? Seems cleaner and would remove the `graffiti` undefined check below.

```suggestion
  async submitTelemetry(payload: { points: Metric[]; graffiti?: string }): Promise<void> {
```

Reply:
Yup, changed it to just making the field optional. (I tried this yesterday and it didn't work but I realized this was due to another bug which is now fixed.)